state.go

```diff
@@ -159,12 +159,14 @@ type StateRunner interface {
 	RunState(ctx *StateContext) (State, bool)
 }
 
-// state information that can be used for restring state from storage
+// StateData is state information that can be used for restroing state from storage
```

Comment:
Crap. Need more coffee in the morning before fixing typos... You know that human brain is working so that it is focusing only on first and last couple of letters in the given word completely ignoring what is in the middle: https://en.wikipedia.org/wiki/Typoglycemia